src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -1592,6 +1645,19 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
                         }
                     }
 
+                    if (dbColumn["is_generated"] === "ALWAYS" && dbColumn["generation_expression"] && supportsGeneratedColumns) {
+                        // In postgres there is no VIRTUAL generated column type
+                        tableColumn.generatedType = "STORED";
```

Comment:
Is there a case where a `VIRTUAL` might be attempted here?

Reply:
This just makes sure it is marked as a generated column. We don't have a boolean switch `is_generated_column` or something similar. This is similar to it.